test/functional/services/common/browser.ts

```diff
@@ -480,6 +480,11 @@ export async function BrowserProvider({ getService }: FtrProviderContext) {
       return this.getScrollTop();
     }
 
+    public async setScrollTopKibanaBody() {
+      await driver.executeScript('document.getElementById("kibana-body").scrollTo(0,0)');
+      // return this.getScrollTop();
+    }
+
```

Comment:
In the current implementation I would rather place it in `commonPage.ts`

Browser service may have more general-purpose function, e.g. 
 
```
    public async scrollToElement(elementId: string) {
      await driver.executeScript(`document.getElementById(${elementId}).scrollTo(0,0)`);
    }
```

Reply:
Yes, I had that same thought.  I'll give that a try